src/main/resources/db/changelog/changes/v1.1.0/changelog-v1.1.0.xml

```diff
@@ -14,4 +14,12 @@
   <include file="update-src-organizations-definition.xml" relativeToChangelogFile="true"/>
   <include file="update-drv-purchase-order-line-details-definition.xml" relativeToChangelogFile="true"/>
 
+  <changeSet id="MODFQMMGR-107" author="mweaver@ebsco.com">
+    <comment>Swap the columns in the query_results primary key, to improve performance</comment>
+    <sql>
+      ALTER TABLE query_results DROP CONSTRAINT IF EXISTS query_results_pkey;
```

Comment:
Should we add` if exists ` with Alter table 
Like ALTER TABLE IF EXISTS ??

Reply:
I feel like having it blow up if the table doesn't exist seems like a reasonable thing to do, since the app will be pretty broken if the table isn't there